src/main/scala/mesosphere/marathon/core/task/update/impl/ThrottlingTaskStatusUpdateProcessor.scala

```diff
@@ -0,0 +1,24 @@
+package mesosphere.marathon.core.task.update.impl
+
+import javax.inject.{ Inject, Named }
+
+import mesosphere.marathon.core.task.update.TaskStatusUpdateProcessor
+import mesosphere.util.CapConcurrentExecutions
+import org.apache.mesos.Protos.TaskStatus
+
+import scala.concurrent.Future
+
+object ThrottlingTaskStatusUpdateProcessor {
+  /**
+    * A tag used for dependency injection to disambiguate the dependencies of this processor from
+    * other instances with the same type.
+    */
+  final val dependencyTag = "ThrottlingTaskStatusUpdateProcessor"
+}
+
+private[core] class ThrottlingTaskStatusUpdateProcessor @Inject() (
```

Comment:
Please not. What you suggest means that the `@Named` annotations have a dependency on the wiring stuff. That's kind of the opposite that you want to achieve by DI.

Yes, I would like to wire without using `@Named` altogether but with our current DI situation I do not know how.